sdk/js/src/Client.ts

```diff
@@ -235,6 +231,10 @@ export class DevCycleClient<
             }
         }
 
+        this.resolveOnInitialized(this)
```

Comment:
why move where this is happening? before it only happened if the config fetching process didnt throw